student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(age, name = 'unknown', parent_permision: true)
+    super(age, name, parent_permision)
+    @classroom = classroom
+  end
```

Comment:
- [ ] classroom should be included in the arguments of the initialize method as well